repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "ReposeFakeKeystonev2",
-  "version": "0.1.0",
-  "description": "Mock Keystone Service",
-  "readme": "README.md",
-  "author": "Open Repose Team <ReposeCore@Rackspace.com> (http://www.openrepose.org/)",
+  "name": "service-mocks",
+  "version": "1.0.0",
+  "description": "Simple services that mock the behavior of more complex services like OpenStack Identity.",
+  "author": "Damien Johnson <damien.johnson@rackspace.com>",
```

Comment:
I'm not sure we want to change the name, major version, description, and author on this version. Also the readme got removed.